packages/generate-coa-pdf-template/package.json

```diff
@@ -44,8 +44,7 @@
     "test:staged": "jest --runInBand --onlyChanged",
     "scan:deps": "audit-ci --config ../../audit-ci.json",
     "create-example": "node ./utils/create-example.js",
-    "create-interfaces": "node ./utils/create-schema-interfaces.js https://schemas.s1seven.com/coa-schemas/v0.1.0/schema.json ./src/types/schemaTypes.ts",
-    "create-interfaces-interactive": "node ../../utils/create-schema-interfaces-interactive.js ./src/types/schemaTypes.ts"
```

Comment:
You did not need to remove `create-interfaces-interactive`, as you were using the `utils` located in the root

Reply:
My bad, I thought it was outdated, but looking back at it I see I was wrong. I've added it back in.